utils/ffmpegvideodecode/ffmpeg_video_dec.cpp

```diff
@@ -724,6 +747,30 @@ void FFMpegVideoDecoder::InitOutputFrameInfo(AVFrame *p_frame) {
     }
 }
 
+/**
+ * @brief Flush decoder with decoding of last frames
+ * 
+ * @return int 1: success 0: fail
+ */
+int FFMpegVideoDecoder::FlushDecoder() {
+    AVPacket pkt = { 0 };
+    if (no_multithreading_) {
+        DecodeAvFrame(&pkt, dec_frames_[av_frame_cnt_]);
+        int num_frames_to_display = decoded_pic_cnt_;
+        while (num_frames_to_display) {
+            RocdecParserDispInfo dispInfo = { 0 }; // don't care about this as this will be igonored
```

Comment:
Please modify the comment for this line to be similar to line 466. Also change `{ 0 }` -> `{0}`